.eslintrc.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     "browser": false
   },
   "parserOptions": {
-    "ecmaVersion": 6
+    "ecmaVersion": 11
```

Comment:
had to bump this as I used object spread

Reply:
Just a word of historical knowledge...

The team traditionally avoided bumping the ecmaVersion beyond what was safe to use for the minimum version of Node we support. This was an additional layer of protection in case a language feature ended up used on a path that was not well tested for some reason so we would not see a crash in our CI for that Node version. As such, usually the test eslintrc got bumped as far as needed to test various language features and the standard one lagged behind. Spread should be covered regardless, I'm guessing ecmascript 9 or 10 is what matches up to Node 12+ mostly.

In reality, the risk of this is likely pretty low these days. We do have some blind spots testing wise but mostly have pretty good coverage. As such, I'm fine landing this.

Where we could get into trouble is with the usage of `Promise.allSettled`, `??`, and `?.` according to: https://node.green/#ES2020. We still need to avoid those in non-test code.

Reply:
yea. we could go up to 19 per node.green.